packages/core/src/logger.ts

```diff
@@ -1,14 +1,16 @@
 import { pino } from 'pino'
 
-export const pinoLogger = pino({
+export const defaultLogger = pino({
   level: (typeof process === 'object' && process.env.LOG_LEVEL) || 'info',
   transport: {
     target: 'pino-pretty',
+    options: {
+      ignore: 'pid,hostname',
+      hideObject: !!process.env.LOG_COMPACT,
+    },
   },
 })
 
-export const defaultLogger = pinoLogger.child({ app: 'chopsticks' })
```

Comment:
no I want this. otherwise I can't distinguish the logs from chopsticks library vs logs from other code in supx